homeassistant/components/tilt_pi/config_flow.py

```diff
@@ -0,0 +1,71 @@
+"""Config flow for Tilt Pi integration."""
+
+import logging
+from typing import Any
+
+import aiohttp
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PORT
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .api import TiltPiClient, TiltPiError
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_NAME): str,
+        vol.Required(CONF_HOST): str,
+        vol.Optional(CONF_PORT, default=1880): int,
```

Comment:
Can someone change the port? Can it do SSL? If we usually access the tilt pi thing via a web interface and need that url, consider just asking for the URL. Check `overseerr` on how we then break down the URL into SSL options, port and whatnot